content/integrations/flyte-plugins/openai/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: OpenAI
+weight: 1
+variants: +flyte +serverless +byoc +selfmanaged
```

Comment:
Should we remove the serverless tag first since v2 is not working on the serverless atm?
```suggestion
variants: +flyte +byoc +selfmanaged
```